spec/spec_helper.rb

```diff
@@ -1,7 +1,16 @@
 require "simplecov"
-require 'coveralls'
 
-SimpleCov.formatter = Coveralls::SimpleCov::Formatter
+if ENV["CI"]
+  require "simplecov-lcov"
+
+  SimpleCov::Formatter::LcovFormatter.config do |c|
+    c.report_with_single_file = true
+    c.single_report_path = "coverage/lcov.info"
+  end
+
+  SimpleCov.formatter = SimpleCov::Formatter::LcovFormatter
+end
+
 SimpleCov.start
```

Comment:
Just realized after writing the previous comment that running specs with coverage can still be done locally and there's no need to run them outputting lcov format - nevertheless introducing an environment variable for the lcov output path makes sense since that parameter value has to match between ci.yml value. Basically - still define that variable in ci.yml and use it in the if statement and path value instead of relying on `CI` environment variable.